x-pack/plugins/alerting/server/application/rule/methods/bulk_disable/bulk_disable_rules.test.ts

```diff
@@ -464,12 +464,48 @@ describe('bulkDisableRules', () => {
     );
   });
 
-  test('should skip rule if it is already disabled', async () => {
+  test('should return both rules if one is already disabled and one is enabled when bulk disable is based on ids', async () => {
     mockCreatePointInTimeFinderAsInternalUser({
       saved_objects: [enabledRuleForBulkOps1, disabledRuleForBulkDisable2],
     });
     unsecuredSavedObjectsClient.bulkCreate.mockResolvedValue({
-      saved_objects: [disabledRuleForBulkDisable1],
+      saved_objects: [disabledRuleForBulkDisable1, disabledRuleForBulkDisable2],
+    });
+
+    const result = await rulesClient.bulkDisableRules({ ids: ['id1', 'id2'] });
+
+    expect(unsecuredSavedObjectsClient.bulkCreate).toHaveBeenCalledTimes(1);
```

Comment:
Why `bulkCreate`, shouldnt it be `bulkUpdate`?

Reply:
Because we use `unsecuredSavedObjectsClient.bulkCreate` to update the rule at line 244 in `x-pack/plugins/alerting/server/application/rule/methods/bulk_disable/bulk_disable_rules.ts`. 

We use `bulkCreate` in `bulkEnable, bulkDisable, bulkEdit`. Why we use it here maybe @pmuellr or @JiaweiWu knows 🙈

Reply:
I think, generally, SO "update" means "partial update", and if you want to do a "full update", you use create w/ overwrite: true, or similar.  Which is semantically the same as the ES APIs (confusing, presumably a historical thing).

Very likely this will change when we work on the "resilience" issue for this, as we have to do some OCC work on the TM docs, that we haven't done before - and probably start doing some "partial" updates: https://github.com/elastic/kibana/issues/192207